src/utils/createRandomXArray.ts

```diff
@@ -0,0 +1,67 @@
+/**
+ * Create a random array of numbers of a specific length, with gaussian distribution with a specific mean and standard deviation
+ *
+ * @param options - options
+ * @return - array of floats
+ */
+
+let spare: number;
+let hasSpare = false;
+
+export function createRandomXArray(
```

Comment:
Coudly ou find a better name ? Is this 'normal' distribution ? So createNormanRandomXArray ?